### PR TITLE
feat: adding tracing for Value Entities #9381

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
References https://github.com/lightbend/kalix/issues/9381

### CODE REVIEW

File: sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/telemetry/Telemetry.scala [hunk 1]
```diff
@@ -60,13 +60,25 @@ final case object EventSourcedEntityCategory extends ComponentCategory {
...
 final class Telemetry(system: ActorSystem) extends Extension {
 
-  def traceInstrumentation(componentName: String, componentCategory: ComponentCategory) =
-    if (system.settings.config.getString(TraceInstrumentation.TRACING_ENDPOINT).isEmpty)
+  val logger = LoggerFactory.getLogger(classOf[Telemetry])
+
+  def traceInstrumentation(componentName: String, componentCategory: ComponentCategory) = {
+    val collectorEndpoint = system.settings.config.getString(TraceInstrumentation.TRACING_ENDPOINT)
+    if (collectorEndpoint.isEmpty) {
+      logger.info("Instrumentation disabled. Set to NoOp.")
```
Comment (johanandren) [inline sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/telemetry/Telemetry.scala:74]: debug, we do not want these showing up in user logs on every user service component startup (on every node start/restart in deployed app)

File: sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/valueentity/ValueEntitiesImpl.scala [hunk 2]
```diff
@@ -151,56 +160,64 @@ final class ValueEntitiesImpl(
...
-            case _ => // non-error
-              val action: Option[ValueEntityAction] = effect.primaryEffect match {
-                case DeleteEntity =>
-                  Some(ValueEntityAction(Delete(ValueEntityDelete(pbCleanupDeletedValueEntityAfter))))
-                case UpdateState(newState) =>
-                  val newStateScalaPbAny = service.messageCodec.encodeScala(newState)
-                  Some(ValueEntityAction(Update(ValueEntityUpdate(Some(newStateScalaPbAny)))))
-                case _ =>
-                  None
+          log.debug("Metadata entries [{}].", metadata.entries)
```
Comment (johanandren) [inline sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/valueentity/ValueEntitiesImpl.scala:164]: Invoked on every request, so probably better as trace

**Review by johanandren — COMMENTED** [review]
I think this is fine, some log level feedback

File: sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/telemetry/Telemetry.scala (hunk 1 above)
Comment (franciscolopezsancho) [reply sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/telemetry/Telemetry.scala]: done

File: sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/valueentity/ValueEntitiesImpl.scala (hunk 2 above)
Comment (franciscolopezsancho) [reply sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/valueentity/ValueEntitiesImpl.scala]: done

**Review by johanandren — APPROVED** [review]

**Pull request merged**